modules/ROOT/pages/spike-control-reference.adoc

```diff
@@ -8,45 +8,80 @@ endif::[]
 >s| Policy Name | Spike Control
 >s|Summary      | Smooths API traffic
 >s|Category |
->s| Since Mule Version | 4.1.0
+>s| First Mule version available | 4.1.0
 .1+>.^s| Returned Status Codes
 |
 |===
 
-In Mule Runtime 4.1.0 and later, the Spike Control Policy is provided for smoothing API traffic. The policy ensures that within any given period of time, no more than the maximum configured requests are processed.
+//Nahuel--any ret codes to doc here?
```

Comment:
HTTP status code 429 - Too many requests. Request is rejected after specified number of reattempts.